features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql

```diff
@@ -2016,6 +2016,73 @@ CREATE TABLE IDN_OAUTH_PAR (
     PARAMETERS CLOB
 )
 /
+CREATE TABLE IDN_SAML2_SERVICE_PROVIDER (
+    ID INTEGER NOT NULL,
+    ISSUER VARCHAR2(255) NOT NULL,
+    DEFAULT_ASSERTION_CONSUMER_URL VARCHAR2(2048) NOT NULL,
+    NAME_ID_FORMAT VARCHAR2(255) NOT NULL,
+    CERT_ALIAS VARCHAR2(255),
+    REQ_SIG_VALIDATION CHAR(1),
+    SIGN_RESPONSE CHAR(1) NOT NULL,
+    SIGN_ASSERTIONS CHAR(1) NOT NULL,
+    SIGNING_ALGO VARCHAR2(255) NOT NULL,
+    DIGEST_ALGO VARCHAR2(255) NOT NULL,
+    ENCRYPT_ASSERTION CHAR(1),
+    ASSERTION_ENCRYPTION_ALGO VARCHAR2(255),
+    KEY_ENCRYPTION_ALGO VARCHAR2(255),
+    ATTR_PROFILE_ENABLED CHAR(1) NOT NULL,
+    ATTR_SERVICE_INDEX VARCHAR2(255),
+    SLO_PROFILE_ENABLED CHAR(1) NOT NULL,
+    SLO_METHOD VARCHAR2(255),
+    SLO_RESPONSE_URL VARCHAR2(2048),
+    SLO_REQUEST_URL VARCHAR2(2048),
+    IDP_INIT_SSO_ENABLED CHAR(1),
+    IDP_INIT_SLO_ENABLED CHAR(1),
+    QUERY_REQUEST_PROFILE_ENABLED CHAR(1) NOT NULL,
+    ECP_ENABLED CHAR(1) NOT NULL,
+    ARTIFACT_BINDING_ENABLED CHAR(1) NOT NULL,
+    ARTIFACT_RESOLVE_REQ_SIG_VALIDATION CHAR(1),
+    IDP_ENTITY_ID_ALIAS VARCHAR2(255),
+    ISSUER_QUALIFIER VARCHAR2(255),
+    SUPPORTED_ASSERTION_QUERY_REQUEST_TYPES VARCHAR2(255),
+    TENANT_ID INTEGER,
+    VERSION VARCHAR(15) NOT NULL,
+    CREATED_AT TIMESTAMP NOT NULL,
+    UPDATED_AT TIMESTAMP NOT NULL,
+    PRIMARY KEY (ID)
+)
+/
+CREATE SEQUENCE IDN_SAML2_SERVICE_PROVIDER_SEQ START WITH 1 INCREMENT BY 1 NOCACHE
+/
+CREATE OR REPLACE TRIGGER IDN_SAML2_SERVICE_PROVIDER_TRIG
+            BEFORE INSERT
+            ON IDN_SAML2_SERVICE_PROVIDER
+            REFERENCING NEW AS NEW
+            FOR EACH ROW
+            BEGIN
+                SELECT IDN_SAML2_SERVICE_PROVIDER_SEQ.nextval INTO :NEW.ID FROM dual;
+            END
```

Comment:
Let's fix the indentation to align with other queries.

Reply:
Resolved in [Introduce JDBC based persistence for SAML](https://github.com/wso2/carbon-identity-framework/pull/6241/commits/d1e67ce649f7a5d2e7e62f364ce93815f347e7b4)